ert_shared/ensemble_evaluator/ensemble/prefect.py

```diff
@@ -57,14 +57,15 @@ async def _eq_submit_job(self, script_filename):
 
 
 def _get_executor(custom_port_range, name="local"):
-    _, port, _ = find_available_port(custom_range=custom_port_range, reuse_addr=True)
     if name == "local":
         cluster_kwargs = {
             "silence_logs": "debug",
```

Comment:
what is the reason of the removal of the port when running locally?

Reply:
It does (I think), but when running locally we are not restricted bu infrastructure firewall :slightly_smiling_face:

Reply:
Ahh - ok. So it's free to pick whatever port id wants then, which it does when we do not provide a port-number.